README.md

```diff
@@ -109,24 +109,25 @@ script:
 ```
 
 ```yaml
-# .gitlabci.yml with Docker runners
+# .gitlab-ci.yml with Docker runners
 stages:
   - split
 
 split:
   image: jderusse/gitsplit
   stage: split
   cache:
-    key: "$CI_JOB_NAME/$CI_COMMIT_REF_NAME"
+    key: "$CI_JOB_NAME-$CI_COMMIT_REF_NAME"
     paths:
-      - /cache/gitsplit
+      - $CI_PROJECT_DIR/cache/gitsplit
```

Comment:
It should be identical to `cache_dir` in the `.gitsplit.yml`. Unless I missed something

Reply:
Well thats true! But then you need to remove that slash there. What i have done is this:
```- cat .gitsplit.yml.dist | sed "s@CI_PROJECT_DIR@$CI_PROJECT_DIR@g" > .gitsplit.yml```
Because gitlab ci doesn't cache outside the build directory so you will get errors about that.

Reply:
> gitlab ci doesn't cache outside the build directory

Oh, I didn't know that. Is it documented ?

BTW you can use relative path for cache. If so, the script will look at the current working dir. It would avoid the sed part of your script ;-)

Can I ask you to test it? If it's ok, may you mind to updated the PR and change cache location to the project for for other CI configuration?